modules/@apostrophecms/rich-text-widget/ui/apos/components/AposRichTextWidgetEditor.vue

```diff
@@ -1,18 +1,17 @@
 <template>
-  <div class="apos-rich-text-editor">
-    <component
-      :is="menuType"
+  <div>
+    <bubble-menu
+      class="bubble-menu"
+      :tippy-options="{ duration: 100 }"
       :editor="editor"
-      :keep-in-bounds="false"
-      v-slot="{ menu, focused }"
+      v-if="editor"
     >
       <AposContextMenuDialog
+        v-if="editor"
```

Comment:
not necessary, right? the parent has the same condition.

Reply:
removed